src/tests/admins.test.js

```diff
@@ -1 +1,122 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import Admin from '../models/admins';
+import adminSeed from '../seeds/admins';
+
+const mockAdmin = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 61165476,
+  phone: 1234567890,
+  email: 'adminPost@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+const mockAdminEmail = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 51165476,
+  phone: 1234567890,
+  email: 'adminPost@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+const mockAdminDni = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 41165476,
+  phone: 1234567890,
+  email: 'adminPost2@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+
+beforeAll(async () => {
+  await Admin.collection.insertMany(adminSeed);
+});
+
+describe('GET /api/admins', () => {
+  test('should return status 200 and length to be 2', async () => {
+    const response = await request(app).get('/api/admins').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.body.data.length).toBe(2);
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBeFalsy();
+    expect(response.body).toBeDefined();
+    expect(response.body.message).toBeDefined();
+  });
+  test('should return status 404, due to invalid url', async () => {
+    const response = await request(app).get('/api/admin').send();
+    expect(response.status).toBe(404);
+    expect(response.body).toBeDefined();
+  });
+  test('should return status 404, as there are no admins in the DB', async () => {
+    await Admin.collection.deleteMany();
+    const response = await request(app).get('/api/admins').send();
+    expect(response.status).toBe(404);
+    expect(response.body.error).toBe(true);
+    expect(response.body).toBeDefined();
+    expect(response.body.data.length).toBe(0);
+    expect(response.body.message).toBeDefined();
+  });
+});
+
+describe('GETBYID /api/admins/', () => {
+  test('should return status 200', async () => {
+    await Admin.collection.insertMany(adminSeed);
+    const response = await request(app).get('/api/admins/646abc7bb96405301276e34f').send();
```

Comment:
Maybe you should use a function to save the first admin ID and use it instead of putting the ID in the url  because if you change the database this test is going to fail

Reply:
the DB will not change during the test, thats why it can be hardcoded